production/helm/loki/Chart.yaml

```diff
@@ -1,13 +1,25 @@
-apiVersion: "v1"
-# This Chart was moved to https://github.com/grafana/helm-charts
-deprecated: true
+apiVersion: v2
 name: loki
-version: 2.1.1
-appVersion: v2.0.0
-kubeVersion: "^1.10.0-0"
-description: "DEPRECATED Loki: like Prometheus, but for logs."
-home: https://grafana.com/loki
-icon: https://raw.githubusercontent.com/grafana/loki/master/docs/sources/logo.png
+description: Helm chart for Grafana Loki in simple, scalable mode
+type: application
+appVersion: 2.6.1
+version: 3.0.0
```

Comment:
is this version the chart version?

Reply:
correct. this chart will be published under `grafana/loki`, which is currently [this](https://github.com/grafana/helm-charts/tree/main/charts/loki) chart (version 2.13.3 currently)

Reply:
got it